_scripts/runDockerRun.sh

```diff
@@ -8,8 +8,9 @@ cd "${BASH_SOURCE%/*}/../"
 
 # The ARTEFACTS variable set by .travis.yml cannot expand
 # variables so we do that here
-ARTEFACTS=$(echo $ARTEFACTS)
+ARTEFACTS=$(echo ${ARTEFACTS:-})
 
+artefacts=""
```

Comment:
leftover?

Reply:
No, this is necessary to avoid the "use before initialised" error that otherwise would occur on line 26 when used outside of CI.